scripts/code-web.js

```diff
@@ -65,8 +83,9 @@ const server = http.createServer((req, res) => {
 	}
 });
 
-server.listen(PORT, () => {
-	console.log(`Web UI available at ${SCHEME}://${AUTHORITY}`);
+server.listen(LOCAL_PORT, () => {
+	console.log(`Operating location at http://0.0.0.0:${LOCAL_PORT}`);
```

Comment:
I would only print this if the local port is different from the remote? Otherwise a bit confusing